doc/uml/Class_Diagram

```diff
@@ -0,0 +1,49 @@
+@startuml
+class Data{
+-Idenitifer
+-Content
+}
+
+class Tag{
+-Identifier
```

Comment:
In general, the UML spec says that field names/methods should be written likeThisInAlternatingCase.

https://www.cs.uct.ac.za/mit_notes/software/htmls/ch05s09.html
> Names of attributes and operations start with a lower-case letter, but may have capital letters to indicate where the spaces would be, if the name would normally have spaces.